packages/babel-preset-env/src/normalize-options.js

```diff
@@ -1,17 +1,15 @@
-//@flow
```

Comment:
Why this change?

Reply:
In one moment of refactoring, support of `flow` created some serious problems and I just disabled it. Maybe in the final version, I'll revert it.